ct/hoarder.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2024 tteck
+# Author: MickLesk (Canbiz) & vhsdream
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+function header_info {
+clear
+cat <<"EOF"
+    __  __                     __
+   / / / /___  ____ __________/ /__  _____
+  / /_/ / __ \/ __ `/ ___/ __  / _ \/ ___/
+ / __  / /_/ / /_/ / /  / /_/ /  __/ /
+/_/ /_/\____/\__,_/_/   \__,_/\___/_/
+
+EOF
+}
+header_info
+echo -e "Loading..."
+APP="Hoarder"
+TAGS="bookmark;links"
+var_disk="8"
```

Comment:
Better make it 10GB, my HDD is at > 90% after the default installation

Reply:
![image](https://github.com/user-attachments/assets/64baa914-3268-411a-820d-3f1c42bec8ae)
This is mine, I'm not sure what would be taking up so much space, although I'll admit this screenshot is not from this exact version.

Reply:
I will test in another branch to confirm 😄

Reply:
@MickLesk @havardthom Here is what my LXC is using in terms of resources immediately after build:
![hoarderfreshinstall](https://github.com/user-attachments/assets/a45d6744-211f-4bbc-97ff-ecf8f7222016)

So about 4GB. I think leaving it at 8 is fine.